manubot/cite/tests/test_doi.py

```diff
@@ -18,3 +21,34 @@ def test_expand_short_doi_invalid():
 def test_expand_short_doi_not_short():
     with pytest.raises(ValueError, match='shortDOIs start with `10/`'):
         expand_short_doi('10.1016/S0933-3657(96)00367-3')
+
+
+def test_get_doi_csl_item_crosscite():
+    doi = '10.1101/142760'
+    csl_item = get_doi_csl_item_crosscite(doi)
+    assert isinstance(csl_item, dict)
+    csl_item['publisher'] == "Cold Spring Harbor Laboratory"
+
+
+def test_get_doi_csl_item_zotero():
+    """
+    As of 2019-10-25, DOI Content Negotiation (i.e. crosscite) encodes the consurtium
```

Comment:
I'm reviewing old pull requests I missed originally.  There's a typo there that could be cleaned up later: `consurtium` -> `consortium`